dvc/repo/__init__.py

```diff
@@ -224,14 +232,14 @@ def __init__(
 
             self._ignore()
 
-        self.metrics = Metrics(self)
-        self.plots = Plots(self)
-        self.params = Params(self)
+        self.metrics: Metrics = Metrics(self)
+        self.plots: Plots = Plots(self)
+        self.params: Params = Params(self)
```

Comment:
Is this really required?

Reply:
It looks odd, but mypy does complain it cannot determine types in `dvc.commands.plots` when using `self.repo.plots.templates_dir` (I

Reply:
Seems it does not compleain for metrics or params though. Got rid of those.